Source/Fuse.Common/Tests/FuseTest/ObservableCollector.uno

```diff
@@ -55,7 +55,7 @@ namespace FuseTest
 			CleanSubscription();
 			if (_items == null)
 				return;
-			_subscription = _items.Subscribe(this);
+			_subscription = (ISubscription)_items.Subscribe(this);
```

Comment:
...however, in the changelog above, you write that IObservableArray.Subscribe() isn't guaranteed to return an ISubscription. This code assumes it does, which seems like a bad assumption.